v2/recipes/multiprice.md

```diff
@@ -88,17 +88,34 @@ use Drupal\commerce_price\Resolver\PriceResolverInterface;
 class CommerceMulticurrencyResolver implements PriceResolverInterface {
 
   /**
+   * The language manager.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
+  /**
+   * Constructs a new CommerceMulticurrencyResolver object.
+   *
+   * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
+   *   The language manager.
+   */
+  public function __construct(LanguageManagerInterface $language_manager) {
```

Comment:
Working on https://www.drupal.org/node/2831954#comment-11906059
and using the implementation suggested here, the tests won't work with $language_manager construct argument.

Please check my PR for the issue https://github.com/drupalcommerce/commerce/pull/626
And maybe we could get the right solution for how this CommerceMulticurrencyResolver should be done